partiql-tests-data/eval/primitives/operators/bag-operators.ion

```diff
@@ -244,8 +395,8 @@ bagOperators::[
     }
   },
   {
-    name:"OUTER UNION with ORDER BY LIMIT on children and bag op",
-    statement:"(SELECT a, tbl FROM t1 ORDER BY a LIMIT 2) OUTER UNION ALL (SELECT a, tbl FROM t2 ORDER BY a LIMIT 2) ORDER BY a LIMIT 2",
+    name:"SQL UNION with ORDER BY LIMIT on children and bag op",
+    statement:"(SELECT a, tbl FROM t1 ORDER BY a LIMIT 2) UNION ALL (SELECT a, tbl FROM t2 ORDER BY a LIMIT 2) ORDER BY a LIMIT 2",
```

Comment:
It *may* be useful to duplicate these tests to have both OUTER and SQL variants to show that OUTER works with SELECTs as well. This is non-blocking however.